src/containers/CoursePage.jsx

```diff
@@ -1,52 +1,70 @@
 import React from 'react';
-import SessionEditorComponent from '../components/SessionEditorComponent';
-import AnnouncementForm from '../components/AnnouncementForm';
-import FeedbackForm from '../components/FeedbackForm';
+// import { Link } from 'react-router-dom';
 // eslint-disable-next-line
 import DraftCSS from 'draft-js';
 
-import QuestionCard from '../components/QuestionCard';
+import InstuctorComponent from '../components/InstructorComponent';
```

Comment:
InstuctorComponent --> InstructorComponent

Small typo

Reply:
I can't believe that actually all went through linting. I guess I kept typo-ing it.